haskell/nettest/FA1_2Comparison.hs

```diff
@@ -57,7 +57,7 @@ fa1_2ComparisonScenario = uncapsNettest $ do
         { sDefaultExpiry = 1000
         , sLedger = BigMap balances
         , sMintingAllowances = mempty
-        , sIsPaused = False
+        , sPaused = False
```

Comment:
Writing this here for lack of a better place, in this commit (89f6743fe548804e7c1d3233f95d770a95d1e042) the message should be corrected, towards the end it says:
> We do, however, have to be careful and make sure the haskell field names the ligo field names,

which I suppose should be:
> We do, however, have to be careful and make sure **that** the haskell field names **match** the ligo field names,

(maybe the `that` is unnecessary tho)

Reply:
Good catch! Done